settings.py

```diff
@@ -1,5 +1,5 @@
-OUTPUT_DIR = ''
-API_BASE = ''
+OUTPUT_DIR = '/home/jerry/work/cfpb/stub-server/'
+API_BASE = 'http://localhost:8181/'
```

Comment:
Damn, I thought I had changed this. Ughhh.

On Tue, Mar 17, 2015 at 7:49 PM, CM Lubinski <notifications@github.com>
wrote:

> In settings.py
> <https://github.com/cfpb/regulations-parser/pull/264#discussion_r26627865>
> :
>
> > @@ -1,5 +1,5 @@
> > -OUTPUT_DIR = ''
> > -API_BASE = ''
> > +OUTPUT_DIR = '/home/jerry/work/cfpb/stub-server/'
> > +API_BASE = 'http://localhost:8181/'
>
> Reminder about local_settings.py
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cfpb/regulations-parser/pull/264/files#r26627865>.
>



-- 
http://www.google.com/profiles/grapesmoker